style.css

```diff
@@ -593,3 +597,652 @@ form {
   border-radius: 8px;
   color: #fff;
 }
+
+.desktop-image {
+  display: none;
+}
+
+.description2 {
+  display: none;
+}
+
+.offer2 {
+  display: none;
+}
+
+.skills2 {
+  display: none;
+}
+
+.Tonic3 {
+  display: none;
+}
+
+.description3 {
+  display: none;
+}
+
+.Tonic4 {
+  display: none;
+}
+
+.offer4 {
+  display: none;
+}
+
+.description4 {
+  display: none;
+}
```

Comment:
- [ ] One of the best practices is avoiding code redundancy, please consider collecting all the similar styling by either creating one class and sharing the class name with those HTML elements or collecting these classes in one group.

### Method 1
```css
.display-none {
    display: none;
}
```

### Method 2
```css
.desktop-image,
.description2,
.offer2,
..... {
  display: none;
}
```

Probably the first method is more preferable among best practices 👍🏼